recipes/usgscsm/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "usgscsm" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3a9617288cf70a0d3752a1e81eec9adacabf59092f2069e8635f5d91ce77d9d1
+
+build:
+  number: 0 
+  skip: True  # [win]
```

Am I correct in guessing this package only contains shared libraries? If so, it might be useful to [declare `run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=run_exports#export-runtime-requirements):
```suggestion
  skip: True  # [win]
  run_exports:
    - {{ pin_subpackage('usgscsm', max_pin='x.x') }}
```
You might need to change `x.x` to `x` or `x.x.x` depending on the ABI stability you expect.

---

You're correct; this is a shared library, specifically a plugin.

From poking around about `run_exports` (the docs are not super useful, unfortunately), it seems to be used for when you want to impose a build + run requirement downstream to other users? This recipe is a plugin. Therefore, no one should include it as part of the building process. 

Correct if I'm wrong.

---

Ah okay, yes you understand correctly and if most people aren't explicitly linking against this library `run_exports` isn't needed. LGTM 👍